src/main/drivers/sound_beeper.c

```diff
@@ -123,7 +123,7 @@ void beeperInit(const beeperDevConfig_t *config)
             IOInit(beeperIO, OWNER_BEEPER, 0);
             IOConfigGPIO(beeperIO, config->isOpenDrain ? IOCFG_OUT_OD : IOCFG_OUT_PP);
         }
-        systemBeep(false);
+        systemBeep(true);
```

Comment:
Is this necessay?

Reply:
Ah, I see now that's how we hacked it :)

Can this be done from the BF settings instead?

Reply:
We need to keep BF changes to a minimum, because it is a merge hell otherwise when we upgrade to a newer BF.

Reply:
Maybe @woutervdhof has some intuition on whether or not this should be possible to do from BF settings?

Reply:
Found a configuration option to invert the systemBeep, testing now

Reply:
Success